frontend/src/views/WheresMyGene/components/GeneSearchBar/components/QuickSelect/index.tsx

```diff
@@ -162,6 +163,14 @@ export default function QuickSelect<
 
   const classes = useStyles();
 
+  const toggleDropdownAndUpdateItemOptions = () => {
+    setOpen(!open);
```

Comment:
To be safe, let's do this instead:

```suggestion
    setOpen((open) => !open);
```

source: https://react.dev/learn/queueing-a-series-of-state-updates#updating-the-same-state-multiple-times-before-the-next-render